build-tools/automation/azure-pipelines.yaml

```diff
@@ -493,6 +466,115 @@ stages:
         targetPath: $(Build.ArtifactStagingDirectory)
       condition: always()
 
+  # Check - "Xamarin.Android (Test Build Tasks Mac)"
+  - job: mac_msbuild_tests
+    displayName: Build Tasks Mac
```

Comment:
Maybe the naming could be better? An idea:
```
# Check - "Xamarin.Android (MSBuild Tests Mac)"
...
displayName: MSBuild Tests Mac
```

Reply:
The display name for the 'stage' is `Test`, and that is always prefixed to the 'job' display name so at the moment we're a bit stuck in our naming options. We could change these to `Test MSBuild Tests Mac` and `Test MSBuild Device Tests Mac`, or maybe even `Test MSBuild Mac`?

Reply:
Ok, `Test MSBuild Mac` and `Test MSBuild for Devices` might be a little better.